src/full/Agda/Interaction/Options/Warnings.hs

```diff
@@ -421,7 +421,7 @@ warningNameDescription = \case
   UnsolvedMetaVariables_           -> "Unsolved meta variables."
   UserWarning_                     -> "User-defined warning added using one of the 'WARNING_ON_*' pragmas."
   WithoutKFlagPrimEraseEquality_   -> "`primEraseEquality' usages with the without-K flags."
-  WrongInstanceDeclaration_        -> "Terms marked as eligible for instance search should end with a name."
+  WrongInstanceDeclaration_        -> "Terms marked as eligible for instance search should be of type {Γ} → C vs, where C evaluates to a postulated name or the name of a data or record type."
```

Comment:
What about `{{x : A}}` domains? Should be fine for recursive instance search.
So maybe the description is: 
> ... `Γ → T` where `Γ` does not contain visible domains and `T` evaluates to ...

Reply:
Actually, at this very location not the full warning text is given, just a short summary what the warning is for.  This was probably wrong before you started changing it.

My suggestion for reformulation thus does not apply here, but at the other locations.

(Here, the formulation should be shortened.)